test/test-remote/test_ui.ts

```diff
@@ -275,6 +278,73 @@ suite("test_ui_with_headless_browser", function () {
     }
   });
 
+  test("test_grafana_datasource_proxy_loki_get_labels", async function () {
+    assert(COOKIES_AFTER_LOGIN);
+
+    const cookie_header_value = COOKIES_AFTER_LOGIN.map(
+      c => `${c.name}=${c.value}`
+    ).join("; ");
+
+    const url = `https://system.${CLUSTER_NAME}/grafana/api/datasources/proxy/2/loki/api/v1/label`;
```

Comment:
needs a code comment: this uses the fixed order of data sources in the Grafana data sources config:

- 0 is cortex
- 1 is loki
- 2 is cortex alertmanager